parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -120,6 +120,9 @@ logs_to_events "AKS.CSE.installNetworkPlugin" installNetworkPlugin
     logs_to_events "AKS.CSE.downloadKrustlet" downloadContainerdWasmShims
 {{- end }}
 
+# By default, never reboot new nodes.
```

Comment:
@paulgmiller pretty sure this has been broken for a while :/ only requested review as fyi, but MIG still needs this.

Reply:
at least people are trying to use MIG now :P

Reply:
also this won't be required for next gen gpu as mentioned below yay